Hudl.Ffmpeg.Tests/packages.config

```diff
@@ -2,6 +2,6 @@
 <packages>
   <package id="OpenCover" version="4.6.519" targetFramework="net45" />
   <package id="xunit" version="1.9.2" targetFramework="net40" />
-  <package id="xunit.runner.console" version="2.1.0" targetFramework="net45" />
+  <package id="xunit.runner.console" version="2.0.0" targetFramework="net45" />
```

Comment:
What still requires an older version of NuGet? I know we require a newer version in various places, so I'm curious if we can make any necessary changes so this can stay on the latest, too.